src/renderer/components/drawer/drawer.scss

```diff
@@ -80,6 +80,17 @@
       padding-right: $padding;
     }
 
+    .drawer-title-text .Icon {
+      visibility: hidden;
+    }
+
+    .drawer-title-text:hover .Icon {
+      font-weight: normal;
+      flex-grow: 2;
+      visibility: visible;
+      box-shadow: none;
+    }
```

Comment:
Some fine-tuning needed to make sure our copy icon looks good. The main thing here is to replace `visibility` prop with `opacity` to make it possible to animate transition on hover.


```
.drawer-title-text .Icon {
  opacity: 0;
  font-weight: normal;
  margin-left: 8px;
}
 
.drawer-title-text:hover .Icon {
  opacity: 1;
  transition: opacity 250ms;
}
```